docs/src/running-validator/validator-start.md

```diff
@@ -323,15 +323,23 @@ the validator to ports 11000-11020.
 
 The `--limit-ledger-size` parameter allows you to specify how many ledger
 [shreds](../terminology.md#shred) your node retains on disk. If you do not
-include this parameter, the validator will keep the entire ledger until it runs
-out of disk space.
-
-The default value attempts to keep the ledger disk usage under 500GB. More or
-less disk usage may be requested by adding an argument to `--limit-ledger-size`
-if desired. Check `solana-validator --help` for the default limit value used by
-`--limit-ledger-size`. More information about
-selecting a custom limit value is [available
-here](https://github.com/solana-labs/solana/blob/36167b032c03fc7d1d8c288bb621920aaf903311/core/src/ledger_cleanup_service.rs#L23-L34).
+include this parameter, the validator will keep all received ledger data
+until it runs out of disk space. Otherwise, the validator will continually
+purge the oldest data once to stay under the specified `--limit-ledger-size`
+value.
+
+The default value attempts to keep the blockstore (data within the rocksdb
+directory) disk usage under 500GB. More or less disk usage may be requested
+by adding an argument to `--limit-ledger-size` if desired. More information
+about selecting a custom limit value is [available
+here]https://github.com/solana-labs/solana/blob/aa72aa87790277619d12c27f1ebc864d23739557/core/src/ledger_cleanup_service.rs#L26-L37).
+
+Note that the above target of 500 GB does not account for other items that
+may reside in the `ledger` directory, depending on validator configuration.
+These items may include (but are not limited to):
+- Persistent accounts data
+- Persistent accounts index
+- Snapshots (both full and/or incremental)
```

Comment:
The old comments don't call out that other stuff gets lumped in `ledger`. I called those out specifically as those are the largest items that I'm aware of that might end up under `ledger`.

That being said, a new operator would probably like to have some idea about the size of these. The aspects that make this tricky are that:
1. The size of snapshots / accounts / accounts index will vary by cluster
2. The size of above items all grow as number of accounts grow
3. Other config options could make these vary (ie number of snapshots to retain)

As a result, I'm reluctant to put a static value as it will become out of date. Even if we choose a number for each cluster, those numbers will get out of date (even if we over-estimate).

Idk, maybe we should overestimate and have a recurring chore to update numbers on some interval. Curious to here other's thoughts

Reply:
I think it's good to clarify what is included in the 500 GB and what is not, but I think ultimately validator owners would like to know how much storage size is considered enough.

Do we also want to provide size estimations for the items that you list above?

Reply:
> but I think ultimately validator owners would like to know how much storage size is considered enough.

Agreed, but again, I'm worried about this information becoming out of date. I think not providing the information is arguably better than having outdated information. I don't think there is anything we could point to that would allow someone to get current numbers for each cluster.

But, having something would be nice which is why I included two reviewers to brainstorm on what is otherwise a minor doc update 😉